topics/index.html

```diff
@@ -4,9 +4,14 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz Topics</title>
+    <title>Trying</title>
```

Comment:
Why did you change it to Trying? I require it to be Quiz Topics, please revert it back.